/jds/bin/diff -uprN sgml-common-0.6.3.old/doc/man/install-catalog.8 sgml-common-0.6.3/doc/man/install-catalog.8
--- sgml-common-0.6.3.old/doc/man/install-catalog.8	2009-02-27 17:56:16.326579000 +0000
+++ sgml-common-0.6.3/doc/man/install-catalog.8	2009-02-27 17:57:55.102431000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .\" This manpage has been automatically generated by docbook2man 
 .\" from a DocBook document.  This tool can be found at:
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
@@ -92,6 +93,20 @@ The SGML centralized catalogs
 Eric Bischoff <ebisch@cybercable.tm.fr>
 .PP
 Developper of install-catalog.
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWgnome-xml-share
+Interface stability^Uncommitted
+.TE
+.sp
 .SH "SEE ALSO"
 \fBjw\fR(1)
 .PP
@@ -110,3 +125,4 @@ http://sourceware.cygnus.com/docbook-too
 the home page of the DocBook tools, a compendium of
 all tools necessary to process DocBook files, including
 the SGML-common stuff.
+...\" LSARC 2005/734 GNOME For Nevada
